AzureCalling/External/Authentication/AADAuthHandler.swift

```diff
@@ -7,18 +7,27 @@ import Foundation
 import MSAL
 
 enum AADAuthStatus {
-    case noAuthRequire
-    case waitingAuth
+    case noAuthRequired
+    case unauthorized
+    case authorizing
     case authorized
 }
 
+enum AADAuthError: Error {
+    case msalNotConfigured
+    case noApplicationContext
+    case noMsalAccount
+    case missingAuthToken
+    case jsonParsingFailed
+    case badImageData
+}
+
 class AADAuthHandler {
```

Comment:
As this class is now dealing with AAD + Graph API, I think we can rename this class or can create separate class for Graph API.